packages/pwa-kit-create-app/scripts/trim-extensions.js

```diff
@@ -8,20 +8,21 @@
 const fs = require('fs')
 const parser = require('@babel/parser')
 const traverse = require('@babel/traverse').default
-const generate = require('@babel/generator').default
 const path = require('path')
 const pluginConfig = require('../assets/plugin-config')
-const {execSync} = require('child_process')
 
 const removeComponentCandidates = new Set() // List of files that are candidates for removal, as a result of trimming.
 const SEPARATOR = path.sep // Use OS-specific path separator
 const COMPONENT_SCAN_PATHS = [
-    `${SEPARATOR}src${SEPARATOR}components${SEPARATOR}`,
-    `${SEPARATOR}src${SEPARATOR}pages${SEPARATOR}`,
-    `${SEPARATOR}src${SEPARATOR}hooks${SEPARATOR}`,
-    `${SEPARATOR}src${SEPARATOR}routes.tsx`,
-    `${SEPARATOR}config${SEPARATOR}constants.js`
+    path.join(SEPARATOR, 'src', 'components', SEPARATOR),
+    path.join(SEPARATOR, 'src', 'pages', SEPARATOR),
+    path.join(SEPARATOR, 'src', 'hooks', SEPARATOR),
+    path.join(SEPARATOR, 'src', 'routes.tsx'),
+    path.join(SEPARATOR, 'config', 'constants.js')
 ]
+const SINGLE_LINE_MARKER = 'sfdc-extension-line'
+const BLOCK_MARKER_START = 'sfdc-extension-block-start'
+const BLOCK_MARKER_END = 'sfdc-extension-block-end'
```

Comment:
these are the special comment markers, in which:
```
// sfdc-extension-line SFDC_EXT_FeatureA removes the next line if SFDC_EXT_FeatureA is turned off
const featureA = import('../featureAComp')

// sfdc-extension-block-start SFDC_EXT_FeatureA removes the code block until the block end marker is encountered
const featureA = ...
const funcA = ...
// sfdc-extension-block-end SFDC_EXT_FeatureA
```

Reply:
When I see `sfdc-extension-block-start` I immediately thing, what is an extension? I'm wondering if we can be more literal. 

```
// @pwa-kit-create-app:gen:if featureX:start
function foo() {
  ...
}
// @pwa-kit-create-app:gen:if featureX:end
```

NOTE: This isn't merge blocking, just something that we might change later before release hopefully

Reply:
And as we discussed the `@` symbol might give people a more comfortable feeling because it looks like an annotation more in the language specific definition like typescript, etc